automation_tools/__init__.py

```diff
@@ -897,6 +912,16 @@ def vm_create():
             host=env['vm_ip'],
         )
 
+    # Execute setup_ddns only if not using VLAN Bridges.
+    if os.environ.get('BRIDGE') is None:
```

Comment:
This could be a single if with three conditions:

```python
if os.environ.get('BRIDGE') is None and 'DDNS_HASH' in os.environ and 'DDNS_PACKAGE_URL' in os.environ:
    ...
```